### DISCUSSION TITLE
Data entry problem when using Repeater

### DISCUSSION THREAD

**Discussion by HudhaifaMoha** [filamentphp/filament#14118] [Help]
### Package

Form builder

### Package Version

3.2.98

### How can we help you?

 public static function form(Form $form): Form
    {
        return $form
            ->schema([
                Repeater::make('attributes') 
                    ->schema([
                        TextInput::make('name')
                            ->required(),
                            
                    ])
                    ->required(),
            ]);
    }
    
    اريد ادخال صفات مرة واحدة وكل واحدة في سجل 
    SQLSTATE[HY000]: General error: 1364 Field 'name' doesn't have a default value (Connection: mysql, SQL: insert into `attributes` (`updated_at`, `created_at`) values (2024-09-02 10:43:21, 2024-09-02 10:43:21))

ويظهر لي الخطاء انه لايوجد الحقل والحقل موجود في قاعدة اليانات ؟

**Comment by leandrocfe** [comment]
Did you add `attributes` as json column in your table?

https://filamentphp.com/docs/3.x/forms/fields/repeater#overview

_We recommend that you store repeater data with a JSON column in your database. Additionally, if you're using Eloquent, make sure that column has an array cast._
